src/models/UserModel.js

```diff
@@ -2,15 +2,20 @@ import { dbConnection } from "../config/database.js";
 import logger from "../utils/logger.js";
 
 export const createUser = async (firstname, lastname, email, password) => {
+	const hashedPassword = await bcrypt.hash(password, 10);
+	req.hashedPassword = hashedPassword;
 	const query =
 		"INSERT INTO users (firstname, lastname, email, password) VALUES ($1, $2, $3, $4) RETURNING *";
-	const values = [firstname, lastname, email, password];
+	const values = [firstname, lastname, email, hashedPassword];
+
+	next();
```

Comment:
remove this.

Reply:
if i remove the values array what does the result now return as the values for the query above?